server/views/pages/recommendations/editDateOfBirth.njk

```diff
@@ -10,24 +10,20 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
             {% include '../../partials/error-summary.njk' %}
-
-            {% set html %}
-                <p class="govuk-body">This will update PPUD.  It will not update NOMIS.</p>
-            {% endset %}
-            {{ govukNotificationBanner({
-                html: html,
-                classes: "govuk-notification-banner--info govuk-body"
-            }) }}
             <h1 class='govuk-heading-l'>{{ pageHeadings[page.id] }}</h1>
-
             <form novalidate method="post">
                 <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-
                 <fieldset class="govuk-fieldset" role="group"
                           aria-describedby="date-of-birth-hint appointment-time-hint{% if errors.dateOfBirth %} error{% endif %}">
-
-                    <div id="date-of-birth-hint" class="govuk-hint">
-                        For example, 31 3 1980
```

Comment:
This hint should be retained as it's part of the standard GDS date entry component.
It needs to be just above the input.
https://design-system.service.gov.uk/components/date-input

Reply:
![image](https://github.com/ministryofjustice/make-recall-decision-ui/assets/104494371/35b82aba-ae1c-4379-9a60-7fa3d9dbba16)
Done!